File: Data Structures & Algorithms/eating-bananas/submission-0.go
func minEatingSpeed(piles []int, h int) int {
	l,r := 1,0
	for _, val := range piles {
		if val > r {
			r = val
		}
	}
	output := r

	for l <= r {
		mid := (l + r) / 2
		totalTime := 0
		for _, pile := range piles {
			totalTime += int(math.Ceil(float64(pile) / float64(mid)))
		}

		if totalTime <= h {
			output = mid
			r = mid - 1
		} else {
			l = mid + 1
		}
	} 
	return output
}

File: Data Structures & Algorithms/pow-x-n/submission-0.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        output = x
        if n > 0:
            for i in range(n - 1):
                output = output * x
            return output
        elif n < 0:
            n = -n
            for i in range(n+1):
                output = output / x
            return output
        else:
            return 1